src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/container_service_client.py

```diff
@@ -79,6 +79,7 @@ class ContainerServiceClient(MultiApiClientMixin, SDKClient):
             'open_shift_managed_clusters': '2018-09-30-preview',
             'container_services': '2017-07-01',
             'managed_clusters': '2019-10-01',
```

Comment:
dont you need to remove this line?

Reply:
Yes, I think you are right - will update